app/assets/javascripts/darkswarm/controllers/group_page_controller.js.coffee

```diff
@@ -19,3 +19,4 @@ Darkswarm.controller "GroupPageCtrl", ($scope, group_enterprises, Enterprises, M
 
   $scope.map = angular.copy MapConfiguration.options
   $scope.mapMarkers = OfnMap.enterprise_markers visible_enterprises
+  $scope.embedded_layout = $location.absUrl().indexOf("embedded_shopfront=true") != -1 
```

Comment:
So this part of the query is set in the iframe, but does it persist when you navigate? I would think that looking for the CSS class of embedded pages would be more reliable, but I'm not sure.

Reply:
It's also interesting that `$location` has `search` for the hash path after `#`, but not for the original query string. In this case it could be better written as `window.location.search.indexOf("embedded_shopfront=true") != -1`. Just in case, somebody calls their shop `my-embedded_shopfront=true`. ;-)

Reply:
"It's also interesting that $location has search for the hash path after #, but not for the original query string." Yes I see what you mean. Agree. 

"Does it persist when you navigate?" I've tested it fairly extensively and seems to never persist, and I can't see how it would persist. But I could have missed something, is there a specific case where you suspect it might?

Reply:
Oh, I don't have the complete picture on my mind. I thought we want that information to persist if people are browsing. Or do all links open in a new tab?

Reply:
Oh I see. Yes actually it looks like all links do in fact open in a new tab. So that means there's no need for the information to persist I guess? I can still change it to look for the CSS class if desired though.